sys/dev/etherswitch/e6000sw/e6000sw.c

```diff
@@ -260,8 +260,9 @@ e6000sw_probe(device_t dev)
 		 */
 		resource_int_value(device_get_name(sc->dev),
 		    device_get_unit(sc->dev), "is8190", &is_6190);
-	resource_int_value(device_get_name(sc->dev),
-		device_get_unit(sc->dev), "is6190x", &is_6190x);
+	if (resource_int_value(device_get_name(sc->dev),
```

Comment:
If "is6190" is available, but "is6190x" isn't, it will just stop the execution. I'm not exactly sure if MV88E6190 and MV88E6190X are related to each other (without one either one can't work).

Also, return isn't returning any meaningful values, and the function expects an integer.